index.html

```diff
@@ -130,9 +134,10 @@ <h2>About me</h2><img class="right-corner" src="icons/Mask Group.svg" alt="">
           don’t hesitate to contact me.</p>
```

Comment:
Well done on addressing the previous review requirements and making improvements to the about us section. However, it seems that the about me section is still not responsive across different screen sizes. It's crucial to ensure that your design adapts well to various devices for a consistent user experience.

Please carefully review the design and make the necessary updates to your code to ensure that the about me section looks good and remains fully functional in different screen sizes. Pay attention to the layout, spacing, and any overlapping elements that might occur on smaller or larger screens.

[Tsige-s-Personal-Portifolio (1).webm](https://github.com/tsigewossne/portfolio/assets/55840999/b98e2d5d-9e60-4079-a2c4-b05811e24f2c)

Remember, responsiveness is key in creating a user-friendly website. If you have any questions or need further assistance, feel free to ask. Keep up the great work! 👍